opentelemetry-sdk/src/opentelemetry/sdk/propagator/b3_format.py

```diff
@@ -15,7 +15,8 @@
 import typing
 
 import opentelemetry.trace as trace
-from opentelemetry.context.propagation.httptextformat import HTTPTextFormat
+from opentelemetry.propagator.httptextformat import HTTPTextFormat
+from opentelemetry.sdk.trace import Tracer
 
 
 class B3Format(HTTPTextFormat):
```

Comment:
I understand that we want to make propagation simpler to use. But at the lowest, building-block level, I think there should still be something that does not depend on the context, if only a free function `extract_b3_spancontext(get_from_carrier, carrier) -> SpanContext`.

Reply:
Could you elaborate with the specific reason why that has value?

Reply:
I have only the abstract reason that you should always strive to keep dependencies to as few as possible.
I imagine it could make combining multiple extractors simpler too.